histolab/masks.py

```diff
@@ -134,8 +135,17 @@ def _regions(regions: List[Region], n: int = 1) -> List[Region]:
 class TissueMask(BinaryMask):
     """Object that represent the whole tissue area mask.
 
-    The tissue within the slide or tile is automatically detected through a predefined
-    chain of filters."""
+    By default, the tissue within the slide or tile is automatically
+    detected through a predefined chain of filters.
+    However, custom filters could be used, too.
```

Comment:
I would move this part of the docstring from the class and place it within the `__init__`. Furthermore I'd get rid of the last line `However ... `